packages/react-core/src/components/Card/Card.tsx

```diff
@@ -12,20 +12,18 @@ export interface CardProps extends React.HTMLProps<HTMLElement>, OUIAProps {
   className?: string;
   /** Sets the base component to render. defaults to article */
   component?: keyof JSX.IntrinsicElements;
-  /** Modifies the card to include hover styles on :hover */
+  /** @deprecated Modifies the card to include hover styles on :hover */
   isHoverable?: boolean;
-  /** @beta Modifies the card to include hoverable-raised styles on :hover, this styling is included by default with isSelectableRaised */
-  isHoverableRaised?: boolean;
   /** Modifies the card to include compact styling. Should not be used with isLarge. */
   isCompact?: boolean;
   /** Modifies the card to include selectable styling */
   isSelectable?: boolean;
   /** Modifies the card to include selected styling */
   isSelected?: boolean;
-  /** @beta Modifies the card to include selectable-raised styling and hoverable-raised styling */
-  isSelectableRaised?: boolean;
-  /** @beta Modifies the card to include selected-raised styling */
-  isSelectedRaised?: boolean;
+  /** @beta Specifies the selectable styling variant */
+  selectableVariant?: 'legacy' | 'raised';
+  /** @beta Modifies a selectable card to have disabled styling */
```

Comment:
Can we be more explicit here that disabled only applies to non raised cards since we still support both variations.

Reply:
As I have it, a disabled selectable card without the raised variant is basically a normal card with no hover/select styles at all. Still valid?

Reply:
Yeah I think only the disabled raised selectable card has any special styling. The flag probably shouldn't apply anything to a non-selectable non-raised card (or alternatively, maybe remove any select related props from non-raised cards?)